TheMET/TheMET/Network/MetAPI.swift

```diff
@@ -106,42 +108,43 @@ class MetAPI{
             parameters: [ : ]) { result in
                 switch result {
                 case .failure:
-                    completion(nil)
+                    completion(.failure(.invalidUrlComponents))
```

Comment:
This is not a `invalidUrlComponents` error. This is some error provided by the completion of the `networkManager.get` method. Use `networkError` as in above.